### ISSUE TITLE
Configuration Documentation needs improved

### ISSUE THREAD

**Issue by fortepc** [Bjornskjald/miscord#187]
While trying to set up this app, this config isn't getting me what I want, even though I am following the instructions and documentation to the best of my ability to understand. 

```{
  "messenger": {
    "username": "insert_my_username",
    "password": "insert_my_password",
    "forceLogin": false,
    "showUsername": true,
    "boldUsername": true,
    "filter": {
      "whitelist": [ID_OF_FBCHAT, ID_OF_SECONDFBCHAT],
      "blacklist": [],
      "_comment": "If you use one of the properties above, leave the other empty."
    },
    "separateImages": false
  },
  "discord": {
    "token": "TOKEN",
    "guild": "ID_OF_SERVER",
    "category": "",
    "sendNotifications": true,
    "_comment": "Regular notifications will still arrive. This switch disables sending duplicate messages when appending new messsage to existing embed (see issue #71)",
    "noEmbeds": false,
    "renameChannels": false,
    "showEvents": false
  },
  "checkUpdates": true,
  "logLevel": "verbose",
  "custom": {
  "ID_OF_FBCHAT": "ID_OF_DISCORDCHANNEL",
  "ID_OF_SECONDFBCHAT": "ID_OF_SECONDDISCORDSERVER"
}
}
```

This doesn't do at all what I want it to do. What am I doing wrong?

Or am I doing everything right and I need to submit a bug report for it just not working as intended?

**Comment by fortepc** [comment]
Incase it's unclear, 

My goal is to have 2 FB chats, mapped

**Comment by fortepc** [comment]
Incase it isn't clear, my goal is to have two FB chats whitelisted, and mapped to existing channels in the server. Right now every time the program restarts it creates an entirely new channel for the messenger group.

**Comment by Bjornskjald** [comment]
Turned out to be an issue with custom mapping. Fixed in #189.
